serving/docs/lmi_new/user_guides/vllm_user_guide.md

```diff
@@ -78,16 +78,20 @@ OPTION_ROLLING_BATCH=vllm
-| option.max_rolling_batch_prefill_tokens	 | \>= 0.26.0  | No	               | Limits the number of tokens for caching. This needs to be tuned based on batch size and input sequence length to avoid GPU OOM. If you don't set, vLLM will try to find a good number to fit in	                                                                                                                                                                                                     | Default: `None`	       |
-| option.max_model_len                     | \>= 0.26.0  | No	               | the maximum length (input+output) vLLM should preserve memory for. If not specified, will use the default length the model is capable in config.json. in verion like 0.27.0, sometimes model's maximum length could go to 32k (Mistral 7B) and way beyond the supported KV token size. In that case to deploy on a small instance, we need to adjust this value within the range of KV Cache limit.	 | Default: `None`	       |
-| option.load_format	                      | \>= 0.26.0  | No	               | The checkpoint format of the model. Default is auto and means bin/safetensors will be used if found.	                                                                                                                                                                                                                                                                                                | Default: `auto`	       |
-| option.enforce_eager                    | \>= 0.27.0  | No	       | vLLM by default will run with CUDA graph optimization to reach to the best performance. However, in the situation of very less GPU memory, having CUDA graph enabled will cause OOM. So if you set this option to true, we will use PyTorch Eager mode and disable CUDA graph to save some GBs of memory.	                                                                                           | Default: `False`	                 |
-| option.gpu_memory_utilization            | \>= 0.27.0  | No	       | This config controls the percentage of memory to be allocated to PagedAttention. Default to 0.9 (90%). We don't recommend to change this value because this impact the overall GPU memory allocations.                                                                                                                                                                                               | Default: `0.9`	                   |
+| option.quantize	                         | \>= 0.26.0  | Pass Through	 | Quantize the model with the supported quantization methods	                                                                                                                                                                                                                                                                                                                                          | `awq` Default: `None`	 |
```

Comment:
This parameter is called `quantization` in vllm not `quantize`. This might be confusing if we call it a passthrough. Maybe we can add this detail in the description

Reply:
I'll change to LMI and update the description as well

Reply:
updated